### DISCUSSION TITLE
Adding the color bar to the plots

### DISCUSSION THREAD

**Discussion by Jbarrers9740** [NatLabRockies/floris#215] [Q&A] [answered]
![ColorBar](https://user-images.githubusercontent.com/86227863/122732128-de71bd00-d273-11eb-9693-b6bae2f5b497.png)
Hi everyone,

How could I add the color bar (black circle in the picture) to my plots? I do not see any example with this legend, nor any reference in the Floris documentation.

Regards,

**Comment by paulf81** [comment, marked as answer]
Hi!  FLORIS is using matplotlib's pcolormesh to draw these in size the visualize_cut_plane function:
https://matplotlib.org/3.1.1/api/_as_gen/matplotlib.pyplot.pcolormesh.html

The function returns the image handle you can use to add this colorbar.  For example, changing example_00_open_and_vis_floris.py so that the last block of code looks like this:

```
# Plot and show
fig, ax = plt.subplots()
im = wfct.visualization.visualize_cut_plane(hor_plane, ax=ax)
fig.colorbar(im, ax=ax)
plt.show()
```
Adds the color bar

**Reply by Jbarrers9740** [reply]
Oh!

I was determined it was like 'colorbar = True' or similar, and it was much
simpler.
Thank you so much for your help and fast response. I really appreciate it.

Regards,
Josue
